tool/tsh/tsh.go

```diff
@@ -939,6 +955,61 @@ func onBenchmark(cf *CLIConf) {
 		fmt.Fprintln(os.Stderr, utils.UserMessageFromError(err))
 		os.Exit(255)
 	}
+	if cf.BenchExport {
+		var fullPath string
+		timeStamp := fmt.Sprint(time.Now().Format("2006-01-02_15:04:05"))
+		suffix := fmt.Sprint("/latency_profile_" + timeStamp + ".txt")
+		// If path is specified
+		if cf.BenchExportPath != "" {
+			usr, err := user.Current()
+			if err != nil {
+				fmt.Fprintln(os.Stderr, utils.UserMessageFromError(err))
+			}
+			dir := usr.HomeDir
+			// Expand tilde if used
+			if cf.BenchExportPath == "~" {
+				cf.BenchExportPath = dir
+			} else if strings.HasPrefix(cf.BenchExportPath, "~/") {
+				cf.BenchExportPath = filepath.Join(dir, cf.BenchExportPath[2:])
+			}
```

Comment:
user shell is responsible for expanding `~`, all of this shouldn't be necessary

Reply:
If we assume the default is `./profile.txt` this logic is no longer necessary

Reply:
@awly when I use a tilde as a flag value, it doesn't expand without this logic. I get a `No such file or directory` error 

@klizhentas are you saying that the user should not have the option to save the file elsewhere and should always save to the working directory?

Reply:
@quinqu are you quoting the flag value by chance?
```sh
❱ echo ~
/home/awly
❱ echo "~"
~
```
The shell will expand it when unquoted.

Regarding export path - I think @klizhentas meant that it should still be a flag, but the default value should be `./` instead of `~/.tsh/`.

Reply:
I am not quoting the flag value @awly

Reply:
I found the issue, it is because I was using `--path=~/temp` and bash does not expand because it is not the first character of the parameter.